### PR TITLE
Change long forecast response

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -72,7 +70,19 @@ def convert_to_readable_time(time):
...
+def convert_to_celsius(temp):
+    return round((temp - 32) * 5 / 9, 2)
+
+
+def convert_to_percentage(amount):
+    return int(amount * 100)
+
+
+def display(forecast):
+    return "Temp: {}C / {}F, Humidity: {}%, Precip: {}%, Summary: {}".format(convert_to_celsius(forecast.temperature), forecast.temperature, convert_to_percentage(forecast.humidity), convert_to_percentage(forecast.precipProbability), forecast.summary)
```
Comment (mknecht) [inline app/views.py:82]: how does this PR address the fact that for NY the temp is in F and for Europe in C?

Comment (admiral96) [inline app/views.py:82]: That response(format) is what Avner wanted. Regarding issues on temperature units will be on a different PR.
